client/src/pages/Search/AddressSearch.js

```diff
@@ -0,0 +1,30 @@
+import { TextField } from "@mui/material";
+import { usePlacesWidget } from "react-google-autocomplete";
+
+export default function AddressSearch({ updateFields }) {
+  const { ref: googlePlacesRef } = usePlacesWidget({
+    apiKey: process.env.REACT_APP_GOOGLE_API_KEY,
+    onPlaceSelected: (place) => {
+      const { location } = place.geometry;
+      const longitude = location.lng();
+      const latitude = location.lat();
+
+      updateFields({ longitude, latitude });
+    },
+    options: {
+      types: ["geocode", "establishment"],
+      componentRestrictions: { country: "usa" },
+    },
+  });
```

Comment:
We may want to set a starting default value at Hunter. So, something like:
`defaultValue="Hunter College, Park Ave, New York, NY, USA"`

Although our app is hoping to use user location to set the initial coordinates, what if they block access to such a request and/or they just want to search a location (perhaps to see ahead of time what food will available when they plan to go somewhere later in the day)? 

In the case of not being able to get user location, I think showing in the search bar that the initial start is Hunter College could provide the user at least some idea of where they are when they first open the app (because only we know that our starting point is Hunter, that is, if this is still the plan for when we don't get user coordinates). Otherwise, I feel they'll have to zoom in/out and waste some time figuring out where they actually are in NYC and if we can save/shorten the 10 or so seconds they'll spend tinkering with the map to find out where they are, the nicer the overall UX will be. Not a big deal, but just my two cents.

Reply:
Yeah good point, we should default it to hunter regardless of user location.
Also I forgot to mention this but we removed the "request user coordinates" a while back, so theres no concern for that